atomistics/shared/output.py

```diff
@@ -7,8 +7,8 @@ class Output:
     def keys(cls):
         return tuple(field.name for field in dataclasses.fields(cls))
 
-    def get(self, engine, *output: str) -> dict:
-        return {q: getattr(self, q)(engine) for q in output}
+    def get(self, engine, output_keys: str) -> dict:
+        return {q: getattr(self, q)(engine) for q in output_keys}
```

Comment:
I like changing the name from `output` to `output_keys`, but I dislike changing it from `*args` to an iterable. Additionally, this change leaves the type hint invalid.